lib/ts/recipe/emailverification/components/features/emailVerification/index.tsx

```diff
@@ -89,11 +89,11 @@ export const EmailVerification: React.FC<Prop> = (props) => {
     const signOut = useCallback(async (): Promise<void> => {
         const session = Session.getInstanceOrThrow();
         await session.signOut(props.userContext);
-        return SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(undefined, props.history);
+        return SuperTokens.getInstanceOrThrow().redirectToAuth({ history: props.history });
     }, [props.recipe]);
 
     const onTokenInvalidRedirect = useCallback(
-        () => SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(undefined, props.history),
+        () => SuperTokens.getInstanceOrThrow().redirectToAuth({ history: props.history }),
```

Comment:
need to explicitly set redirectBack to false -> internal use of this function should always explicitly set this value to prevent future bugs.